frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts

```diff
@@ -190,7 +190,8 @@ export class CastDragAndDrop implements OnInit {
     this.castPositions = [];
     if (!this.castAPI.hasCast(this.selectedCastUUID)) {
       this.castSelected = false;
-      this.logging.logError("Couldn't find cast: " + this.selectedCastUUID);
+      // The below error means little. After any save the system loses tracf the uuid of the saved cast
```

Comment:
Ack, can remove the commented out code and corresponding comment.

Reply:
I want to leave this comment in. I spent several hours tracking down this error and figuring out why it happened (and also realizing that it was very difficult to fix without some serious logic rewrite). The comment is marking where future work should happen.

Reply:
That makes sense, in that case let's remove the dead code line and just leave a comment in this format:
```
// TODO: After any save the system loses track the uuid of the saved cast, so we need to fix [XYZ].
```

Reply:
Done.

Reply:
Perfect.